Supporting AlphaRegistry and new Registry, new packaging system for contract jsons

vforvalerio87 opened the pull request:
Added support for both AlphaRegistry and the new Registry.

Added a resources directory with migrations to deploy contracts for testing and development purposes; added scripts to package those contracts and drop them in the node_modules directory as symlinks to be used in the dapp source code without modifications.

Modified import statements in the source code to work with separate abi and networks json files for blockchain contracts and token contracts.

Changed references for Registry contracts to AlphaRegistry.

Fixed all links to etherscan so that the href is undefined if the current network name does not match a network that is in our file: before if you tried to run the dapp with any unknown network (ganache-cli) it would simply crash.

Added a dependency that was missing from package.json (underscore).

Updated versions of dependencies that had vulnerabilities in them.

tiero commented on src/components/services.js:
```diff
@@ -27,7 +27,7 @@ class Services extends React.Component {
           this.props.network &&
           <Tag>
-            <a target="_blank" href={`${NETWORKS[this.props.network].etherscan}/address/${address}`}>
+            <a target="_blank" href={this.props.network && typeof NETWORKS[this.props.network] !== "undefined" ? `${NETWORKS[this.props.network].etherscan}/address/${address}` : undefined}>
```
Should be this an helper function in order to re-use it both here and in account.js?

vforvalerio87 commented:
@lcdupree I fixed the function names and rebased from the latest commit in the master branch.
Tested, I get a 403 error on the preflight request when calling "SingularityNET/ExampleService" (CORS) but I see there's a PR for that that was just merged (https://github.com/singnet/snet-daemon/pull/19)
